ag-grid-webcomponent/app_renderer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Implements a cellRenderer for AgGrid to use an Elm application for the cell.
+ *
+ * https://www.ag-grid.com/javascript-data-grid/component-cell-renderer/#cell-renderer-component
+ */
+class AppRenderer {
+  /**
+   * Gets called once before the renderer is used.
+   * Loads the Elm application into a `DocumentFragment`.
+   * @param {*} params Contains encoded user and table data
+   */
+  init(params) {
+    this.eGui = new DocumentFragment();
+
+    const component = window.ElmAgGridComponentRegistry[params.componentName];
```

Comment:
We could show a nice error message that says that the app is not added to the component registry if we can not find it.